e3sm_to_cmip/util.py

```diff
@@ -331,29 +361,6 @@ def get_handler_info_msg(handler):
     return msg
 
 
-def _use_highfreq_handler(var: str, freq: str) -> bool:
-    """
-    Use the high frequency version of a variable handler based on the
-    specified frequency.
-
-    For example, for "pr" with a frequency of "day", use the handler
-    "pr_highfreq.py" instead of "pr.py".
-
-    Parameters
-    ----------
-    var : str
-        The variable.
-    freq : str
-        The frequency.
-
-    Returns
-    -------
-    bool
-        True if high frequency applies to variable, otherwise False.
-    """
-    return var in HIGHFREQ_VARS and freq in FREQUENCIES["high"]
-
-
```

Comment:
"High frequency" related code is no longer used since variable handlers were refactored awhile ago. I removed this code since it's been lingering around.